GOVERNANCE.md

```diff
@@ -1,45 +1,107 @@
+
+Major changes such as feature proposals and organization or process changes should be brought to the PMC or Maintainers’ attention through the [Request for Comments (RFC)](https://github.com/tikv/rfcs) process. For the change to happen, the RFC must earn the supermajority (2/3) votes in the corresponding group.
+
+## Conflict resolution
+
+In general, we prefer that technical issues and other disputes upon which consensus can't be reached are amicably worked out between the persons involved. If a dispute cannot be resolved independently, depending on the nature, the PMC or Maintainers can be called in to resolve the issue by voting based on the [Guiding Principles](/guiding-principles.md) and [CNCF Code of Conduct](https://github.com/tikv/tikv/blob/master/CODE_OF_CONDUCT.md). The same PR can be used or a separate PR can be opened in the concerned repository for voting.
+
+## Adding new projects
+
+New projects can be added to the TiKV organization via GitHub issue discussion in one of the existing SIG, as long as as they adheres to the [CNCF charter](https://www.cncf.io/about/charter/) and the guidelines in this governance. Once sufficient discussions have taken place, the Maintainers will decide whether the new project should be added. The requester needs to create an corresponding RFC for the change to happen, as described in [Decision Making and Voting](#decision-making-and-voting).
 
-### Organization
+## Changes in Governance
 
-* SIG: Special Interest Group, need invitation currently. Members are Contributors who are active in a particular project. The ultimate goal of the group is to train community talents and help them advance to the higher-level Reviewer, Committer and Maintainer.
-* WG:  Working group open for everyone, community members can join without a threshold. The working group is a temporary group established to complete a short-term project, and the project is dissolved when it is completed. A working group can involved in multiple interest groups. After a period of training, excellent community members will be selected into a special interest group for further study.
+Any change in this Governance document, or similar nature of changes to other governance related documents, shall be initiated as an RFC and go through the voting process as described in [Decision Making and Voting](#decision-making-and-voting).
 
-### Internal role:
+## Credits
 
-- Tech Leader: The technical leader of the group controls the technical direction of the entire team. Responsible for formulating study plans, member assessment rules, etc. A team can have multiple Tech Leaders.
-- Please see the community organization chart for the rest of the roles
+The contents of this document are based on <http://oss-watch.ac.uk/resources/meritocraticgovernancemodel> by Ross Gardler and Gabriel Hanganu, and [Kubernetes Governance](https://github.com/kubernetes/community/blob/master/governance.md).
```

Comment:
I would suggest that basing the governance model on Kubernates is inappropriate because it is a well-established, large project, with  many large orgs involved who are materially affected by K8s decisions. TiKV should look to smaller projects for governance models (e.g., Ember, Rust, Linkerd, etc) and choose a more lightweight model.